lib/src/components/servo/service.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:grpc/grpc.dart';
+import 'package:viam_sdk/src/components/servo/servo.dart';
+import 'package:viam_sdk/src/gen/common/v1/common.pb.dart';
+import 'package:viam_sdk/src/gen/component/servo/v1/servo.pbgrpc.dart';
+import 'package:viam_sdk/src/resource/manager.dart';
+import 'package:viam_sdk/src/utils.dart';
+
+class ServoService extends ServoServiceBase {
+  final ResourceManager _manager;
```

Comment:
Why `final`?

Reply:
I want it to not be able to be set again. It may be overkill. I dont _think_ it will stop things from being added to it, just stops it from being pointed to a different object

Reply:
I think `final` makes sense then.